src/main/java/com/techsophy/tsf/account/controller/impl/InternalControllerImpl.java

```diff
@@ -40,6 +42,7 @@ public ApiResponse<UserFormDataSchema> saveUser(UserFormDataSchema internalUserF
             if(headers.containsKey(X_SIGNATURE)) {
                 String headerSign = headers.getFirst(X_SIGNATURE);
                 UserFormDataSchema userFormDataSchema = rsa4096.transform(headerSign, internalUserFormDataSchema);
+                logger.info("userFormDataSchema in internalController");
```

Comment:
this should not be required as the aspect should take care of it, can u check why is it not working